### ISSUE TITLE
Call create issue API from the "Create Issue" dialog

### ISSUE THREAD

**Issue by alansemenov** [enonic/xp#4553]
Depends on #4547, #4551 

When the "Create issue" button is clicked in the dialog, call issue API to create an issue. 

If there was an error creating an issue, show the red error notification.

If the issue was created successfully, close the dialog and show blue notification "New issue <issueId> created successfully.".
